mathesar_ui/src/i18n/languages/en/dict.json

```diff
@@ -24,6 +24,7 @@
   "add_user": "Add User",
   "admin": "Admin",
   "admin_access": "Admin Access",
+  "admin_user_checkbox_help": "Admin users have some additional privileges, but only for high-level Mathesar-specific operations like connecting databases and managing other users. A user's admin status does not affect their access to the data within the connected databases.",
```

Comment:
> A user's admin status does not affect their access to the data within the connected databases.

This text might end up misleading the user. An admin can modify their own access (and the access of others) to connected databases. So, the admin status does affect the access to the data within the connected databases.

Reply:
@pavish do you have a specific change to propose here?